src/index.css

```diff
@@ -0,0 +1,9 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer base {
+  html {
+    font-family: "Montserrat", sans-serif;
```

Comment:
I would suggest using some different font, Montserrat looks big and messy if we have more content.
You can go for Inter, Rubik, Josefin Sans. Btw just a suggestion!